docs/publish.md

```diff
@@ -2582,6 +2582,11 @@ format is:
 ntfy-$topic@ntfy.sh
 ```
 
+If [access control](config.md#access-control) is enabled and `auth-default-access` is set to `deny-all`, e-mail publishing won't work without providing an authorized access token. That will change the format of the e-mail's recipient address to
+```
+ntfy-$topic+$token@ntfy.sh
+```
+
```

Comment:
A little more correct: "If access control is enabled, and the target topic does not support anonymous writes, ...`

Reply:
Updated :)